documentation/modules/exploit/linux/ssh/ssh_erlangotp_rce.md

```diff
@@ -0,0 +1,153 @@
+## Vulnerable Application
+
+Erlang/OTP is a set of libraries for the Erlang programming language.
+
+Prior to versions OTP-27.3.3, OTP-26.2.5.11, and OTP-25.3.2.20, a SSH server may allow an attacker
+to perform unauthenticated remote code execution (RCE).
+
+By exploiting a flaw in SSH protocol message handling, a malicious actor could gain unauthorized access
+to affected systems and execute arbitrary commands without valid credentials. This issue is patched in
+versions OTP-27.3.3, OTP-26.2.5.11, and OTP-25.3.2.20.
+
+A temporary workaround involves disabling the SSH server or to prevent access via firewall rules.
+
```

Comment:
I'm not sure that this is applicable. Or, conversely, this is universally applicable to every vulnerability in every network service.

```suggestion
```